CMakeLists_files.cmake

```diff
@@ -236,6 +236,8 @@ list (APPEND PUBLIC_HEADER_FILES
   opm/autodiff/WellHelpers.hpp
   opm/autodiff/StandardWells.hpp
   opm/autodiff/StandardWells_impl.hpp
+  opm/autodiff/WellInterface.hpp
+  opm/autodiff/StandardWell.hpp
```

Comment:
The _impl files must also be added here, since they are included from the main public header.